Gemfile

```diff
@@ -67,7 +67,7 @@ end
 gem 'rubocop', '~> 0.35.0', require: false
 gem 'blacklight', '~> 5.16'
 gem 'blacklight-spotlight', github: 'pulibrary/spotlight', branch: 'namespace_view_plus_readonly_field_names'
-gem 'jettywrapper', '>= 2.0'
+gem 'solr_wrapper', '~> 0.14.1'
```

Comment:
Yep, just bumped to 0.15 and put in a new PR: https://github.com/pulibrary/pomegranate/pull/114